### PR TITLE
ByteArrayToImageSourceConverter

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `ByteArrayToImageSourceConverter ` <https://github.com/CommunityToolkit/Maui/issues/51> includes:

### CODE REVIEW

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/byte-array-to-image-source-converter.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The `Convert` method returns the supplied `byte[]` `value` converted to an `ImageSource`.
+
+The `ConvertBack` method returns the supplied `ImageSource` `value` converted to a `byte[]`.
+
+## Syntax
+
+### XAML
+
+The `DoubleToIntConverter` can be used as follows in XAML:
```
Comment (jfversluis) [inline docs/maui/converters/byte-array-to-image-source-converter.md:22]:
```suggestion
The `ByteArrayToImageSourceConverter ` can be used as follows in XAML:
```

File: docs/maui/converters/byte-array-to-image-source-converter.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    public ByteArrayToImageSourceConverterPage()
+    {
+        var image = new Image();
+
+		image.SetBinding(
+			Image.SourceProperty,
+			new Binding(
+				nameof(ViewModel.DotNetBotImageByteArray),
+                mode: BindingMode.OneWay,
+				converter: new ByteArrayToImageSourceConverter()));
```
Comment (jfversluis) [inline docs/maui/converters/byte-array-to-image-source-converter.md:57]:
```suggestion
				nameof(ViewModel.DotNetBotImageByteArray),
				mode: BindingMode.OneWay,
				converter: new ByteArrayToImageSourceConverter()));
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Minor things and needs some conflicts resolved

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/byte-array-to-image-source-converter.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        var image = new Image();
+
+		image.SetBinding(
+			Image.SourceProperty,
+			new Binding(
+				nameof(ViewModel.DotNetBotImageByteArray),
+				mode: BindingMode.OneWay,
+				converter: new ByteArrayToImageSourceConverter()));
+
+		Content = image;
```
Comment (jfversluis) [inline docs/maui/converters/byte-array-to-image-source-converter.md:59]:
```suggestion
	image.SetBinding(
		Image.SourceProperty,
		new Binding(
			nameof(ViewModel.DotNetBotImageByteArray),
			mode: BindingMode.OneWay,
			converter: new ByteArrayToImageSourceConverter()));

	Content = image;
```

**Review by jfversluis — APPROVED** [review]

**Comment by PRMerger7** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger9** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Pull request merged**